### DISCUSSION TITLE
twitter patches

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [ReVanced/revanced-discussions#523] [Question] [answered]
Add more patches for twitter which are

viral tweets

viral videos

viral pics

topics to follow

Example below

![IMG_20220823_201942.jpg](https://user-images.githubusercontent.com/38470944/186197115-8e4c356f-f324-4f84-a2c8-48d9be46a343.jpg)

![IMG_20220823_200353.jpg](https://user-images.githubusercontent.com/38470944/186193388-dd5fa27e-b862-485f-89ed-acf53fb98794.jpg)

![IMG_20220823_222150.jpg](https://user-images.githubusercontent.com/38470944/186224282-31dc3f3d-62b6-46bf-8a6b-4f203189f67c.jpg)

![IMG_20220823_222132.jpg](https://user-images.githubusercontent.com/38470944/186224305-fbb317f2-7585-4e58-afae-c8c2082b1597.jpg)

patches to disable above things on twitter

**Comment by oSumAtrIX** [comment, marked as answer]
Explain your suggestion

**Reply by HikaruIchijyo** [reply]
> Explain your suggestion

chk now

**Comment by sertchan** [comment]
Up im agree with this guy follow topic section just annoying. ![Screenshot_20220824-225125_Twitter~2.png](https://user-images.githubusercontent.com/53477192/186510407-cf8ca1f2-4915-463b-b8bf-c0d9081ccb69.png)
